tests_e2e/tests/scripts/modify-agent-publish-config

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
I added tests_e2e/orchestrator/scripts/update-waagent-conf in a previous PR. We can use that script to update waagent.conf instead of repeating this logic across tests.